pandas/core/frame.py

```diff
@@ -4031,7 +4033,25 @@ def drop(
         "mapper",
         [("copy", True), ("inplace", False), ("level", None), ("errors", "ignore")],
     )
-    def rename(self, *args, **kwargs):
+    def rename(
```

Comment:
Change here was to be explicit about what is accepted

Reply:
+1 to being explicit. The only reason we had *args, **kwargs in the first place was to support the ambiguous case.